src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,457 @@
+.pf-c-wizard {
+  --pf-c-wizard--GridTemplateRows: auto auto 1fr auto;
+  --pf-c-wizard--lg--GridTemplateRows: auto 1fr auto;
+  --pf-c-wizard--GridTemplateColumns: 100%;
+  --pf-c-wizard--lg--GridTemplateColumns: 300px 1fr;
+  --pf-c-wizard--Height: 100%;
+  --pf-c-wizard--Width: 100vw;
+  --pf-c-wizard--lg--Height: #{pf-size-prem(762px)}; // Shares height with about modal
```

Comment:
The way that the wizard truncates when the viewport height reduces is different to the about modal. Not sure if the content should remain and the buttons are hidden or vice versa. But I think we should decide here and then create an issue to update the about modal.

<img width="1044" alt="screen shot 2019-02-19 at 10 19 16 am" src="https://user-images.githubusercontent.com/20118816/53025914-e6b15780-342f-11e9-8989-ee1111b9af89.png">

<img width="878" alt="screen shot 2019-02-19 at 10 20 55 am" src="https://user-images.githubusercontent.com/20118816/53025999-13fe0580-3430-11e9-9c43-13db83145c0f.png">

Reply:
fwiw, your screenshot is of the normal modal, not the [about modal](https://pf4.patternfly.org/components/AboutModalBox/examples/).

@kybaker what do you think? This is on the edge case that a person has a super short viewport. 

Here are some screenshots:

## normal height

<img width="1237" alt="screen shot 2019-02-19 at 9 52 46 am" src="https://user-images.githubusercontent.com/35148959/53028292-368e1f80-342c-11e9-8939-fa1623150f94.png">
<img width="512" alt="screen shot 2019-02-19 at 9 53 07 am" src="https://user-images.githubusercontent.com/35148959/53028293-368e1f80-342c-11e9-9bbc-00e9146e0d48.png">

## super short height

<img width="1223" alt="screen shot 2019-02-19 at 9 52 55 am" src="https://user-images.githubusercontent.com/35148959/53028326-4443a500-342c-11e9-9c2d-617f93297454.png">
<img width="512" alt="screen shot 2019-02-19 at 9 53 12 am" src="https://user-images.githubusercontent.com/35148959/53028327-4443a500-342c-11e9-9479-7ffdd27375e5.png">